src/components/Territory.vue

```diff
@@ -103,17 +105,33 @@ export default {
     ...mapActions({
       getTerritory: 'territory/getTerritory',
       resetNHRecords: 'territory/resetNHRecords',
+      checkinTerritory: 'territory/checkinTerritory',
     }),
 
-    async resetNH() {
-      if (window.confirm('Are you sure you want to reset NH records?')) {
+    async checkIn() {
+      if (window.confirm('Ready to check-in the territory?')) {
         this.isLoading = true;
         await this.resetNHRecords(this.id);
         await this.getTerritory(this.id);
```

Comment:
you can prob remove this line because we're no longer staying in the Territory page
```await this.getTerritory(this.id);```